Meta/build-root-filesystem.sh

```diff
@@ -137,11 +137,10 @@ chmod 755 mnt/res/devel/templates/*.postcreate
 echo "done"
 
 printf "creating initial filesystem structure... "
-for dir in bin etc proc mnt tmp boot www mod var/run usr/local usr/Ports usr/bin; do
```

Comment:
(commit message would've been better if it ended with something like "and is no longer needed after 04ba31b8c5ec71 which removed support for loadable kernel modules again.)